zerver/views/report.py

```diff
@@ -72,7 +79,8 @@ def json_report_error(request, user_profile, message=REQ, stacktrace=REQ,
                       ui_message=REQ(validator=check_bool), user_agent=REQ,
                       href=REQ, log=REQ,
                       more_info=REQ(validator=check_dict([]), default=None)):
-
+    # type: (Any, UserProfile, Any, Any, REQ, Any, Any, Any, REQ) -> HttpResponse
+    # TODO: INCOMPATIBLE_TYPES_WITH_REQ_TYPE_OBJECT_AND_REQ
```

Comment:
If we can't figure out another solution, we could just get rid of the no-argument REQ() function model.
```
(zulip-venv) tabbott@zaset:~/zulip$ git grep '=REQ[,)]' 
zerver/decorator.py:    def _wrapped_view_func(request, api_key=REQ,
zerver/decorator.py:    def _wrapped_view_func(request, email=REQ, api_key=REQ('api_key', default=None),
zerver/views/__init__.py:def json_invite_users(request, user_profile, invitee_emails=REQ):
zerver/views/__init__.py:def api_fetch_api_key(request, username=REQ, password=REQ):
zerver/views/__init__.py:def update_active_status_backend(request, user_profile, status=REQ,
zerver/views/__init__.py:def json_refer_friend(request, user_profile, email=REQ):
zerver/views/__init__.py:def add_apns_device_token(request, user_profile, token=REQ, appid=REQ(default=settings.ZULIP_IOS_APP_ID)):
zerver/views/__init__.py:def add_android_reg_id(request, user_profile, token=REQ):
zerver/views/__init__.py:def remove_apns_device_token(request, user_profile, token=REQ):
zerver/views/__init__.py:def remove_android_reg_id(request, user_profile, token=REQ):
zerver/views/messages.py:def render_message_backend(request, user_profile, content=REQ):
zerver/views/report.py:def json_report_error(request, user_profile, message=REQ, stacktrace=REQ,
zerver/views/report.py:                      ui_message=REQ(validator=check_bool), user_agent=REQ,
zerver/views/report.py:                      href=REQ, log=REQ,
zerver/views/streams.py:def add_default_stream(request, user_profile, stream_name=REQ):
zerver/views/streams.py:def remove_default_stream(request, user_profile, stream_name=REQ):
zerver/views/streams.py:def json_rename_stream(request, user_profile, old_name=REQ, new_name=REQ):
zerver/views/streams.py:def json_make_stream_public(request, user_profile, stream_name=REQ):
zerver/views/streams.py:def json_make_stream_private(request, user_profile, stream_name=REQ):
zerver/views/streams.py:def json_stream_exists(request, user_profile, stream=REQ,
zerver/views/user_settings.py:                         full_name=REQ,
zerver/views/users.py:def add_bot_backend(request, user_profile, full_name=REQ, short_name=REQ,
zerver/views/users.py:def create_user_backend(request, user_profile, email=REQ, password=REQ,
zerver/views/users.py:                        full_name=REQ, short_name=REQ):
zerver/views/webhooks/github.py:def api_github_landing(request, user_profile, event=REQ,
zilencer/views.py:def report_error(request, deployment, type=REQ, report=REQ(validator=check_dict([]))):
```